### PR TITLE
build Swiss Narcolepsy Scale page

### PR DESCRIPTION
**Pull request by martindev727**

### CODE REVIEW

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx [hunk 1]
```diff
@@ -1,7 +1,148 @@
...
+        'American Academy of Sleep Medicine. Central disorders of hypersomnolence. In: ',
+      source: 'The International Classification of Sleep Disorders – Third Edition (ICSD-3). ',
+      info: 'Darien, IL: American Academy of Sleep Medicine; 2014.'
+    }
+  ]
+  return (
+    <div className="swiss-narcolepsy-scale">
+      <div className="primary-container">
+        <TopLinkComponent topLinkContent={topLinkContent} />
+        <h1 className="main-page-header">Swiss Narcolepsy Scale</h1>
```
[the remark this answers is not in the document]
Comment (rphovley) [reply src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: Padding above and below most elements on the screen on mobile is incorrect and too large. I circled the top half here but the bottom half also has some problems. You may not have targeted mobile for any of the spacing
screen shot 2018-08-21 at 10 57 13 am

File: src/components/learn-more/learn-more.component.jsx [hunk 2]
```diff
@@ -16,10 +16,17 @@ const LearnMoreComponent = props => {
...
           <SiteLink className="learn-more-link" to={props.learnMoreContent.link}>
-            <div>Learn More</div>
+            <div>{props.learnMoreContent.linkLabel || 'Learn More'}</div>
             <FontAwesomeIcon className="arrow-right" icon="angle-right" />
           </SiteLink>
         </div>
+        { props.learnMoreContent.otherLink && (
+            <div>
+              <SiteLink className="learn-more-other-link" to={props.learnMoreContent.otherLink}>
+                <div>{props.learnMoreContent.otherLinkLabel}</div>
```
Comment (rphovley) [inline src/components/learn-more/learn-more.component.jsx]: font-size should be 14px on mobile

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx [hunk 3]
```diff
@@ -1,7 +1,148 @@
...
+            behavioral symptoms that may be associated with narcolepsy with cataplexy.<sup className="super">1</sup>
+          </p>
+          <div className="learn-more-content">
+            <LearnMoreComponent learnMoreContent={learnMoreContent} />
+          </div>
+        </div>
+      </div>
+      <div className="feature-wrapper">
+        <div className="primary-container">
+          <h2 className="section-header">Features of the SNS</h2>
```
[the remark this answers is not in the document]
Comment (rphovley) [reply src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: Also, font-weight should be `bold` not `normal` on this header

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx [hunk 4]
```diff
@@ -1,7 +1,148 @@
...
+                </p>
+              </div>
+              <div className="feature-item">
+                <div className="feature-name">Assessments:</div>
+                <div className="feature-content">
+                  <p>Measures frequency of 5 potential symptoms<sup className="super">1</sup></p>
+                  <p><b>Q<sup className="sup-bottom">1</sup></b> – Inability to fall asleep</p>
+                  <p><b>Q<sup className="sup-bottom">2</sup></b> – Feeling bad or not well rested in the morning</p>
+                  <p><b>Q<sup className="sup-bottom">3</sup></b> – Taking a nap during the day</p>
+                  <p><b>Q<sup className="sup-bottom">4</sup></b> – Weak knees/buckling of the knees during emotions such as laughing, happiness, or anger</p>
```
Comment (rphovley) [inline src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: Longer lines on mobile should not wrap all the way back. When the line wraps, it should start back in line with the start of the text. Refer to the mobile designs to see what I'm talking about
screen shot 2018-08-21 at 11 05 50 am

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx [hunk 5]
```diff
@@ -1,7 +1,148 @@
...
+              </div>
+              <div className="feature-item">
+                <div className="feature-name">Validation:</div>
+                <p className="feature-content">
+                  In 1 study of patients with narcolepsy with cataplexy, an SNS score &lt;0 was shown to have a sensitivity of 96% and specificity of 98%.<sup className="super">2</sup>
+                </p>
+              </div>
+            </div>
+            <p className="feature-comment">
+            A narcolepsy diagnosis should be established by a sleep specialist using a clinical interview and a nighttime polysomnogram (PSG) followed by a multiple sleep latency test (MSLT) performed under standardized conditions.<sup className="super">3</sup>
```
Comment (rphovley) [inline src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: font-size should be 12px on mobile and 16px on desktop

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx [hunk 6]
```diff
@@ -1,7 +1,148 @@
...
+            <div className="tertiary-header">
+              Two narcolepsy screening tools are now available in 1 app
+            </div>
+            <SiteLink className="link-text" to="Screener App">
+              Learn more about the Narcolepsy Screener App&nbsp;&nbsp;<FontAwesomeIcon className="arrow-right" icon="angle-right" />
+            </SiteLink>
+          </div>
+          <img src={imgAppScreen} alt='Narcolepsy screening tools app' />
+        </div>
+        <div className="app-comment">
```
[the remark this answers is not in the document]
Comment (martindev727) [reply src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: fixed

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx [hunk 7]
```diff
@@ -1,7 +1,148 @@
...
+              <div className="feature-item">
+                <div className="feature-name">Population:</div>
+                <p className="feature-content">
+                  Patients with excessive daytime sleepiness whom the clinician may want to screen for potential narcolepsy with cataplexy<sup className="super">2</sup>
+                </p>
+              </div>
+              <div className="feature-item">
+                <div className="feature-name">Assessments:</div>
+                <div className="feature-content">
+                  <p>Measures frequency of 5 potential symptoms<sup className="super">1</sup></p>
```
[the remark this answers is not in the document]
Comment (martindev727) [reply src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: fixed

File: src/components/learn-more/learn-more.component.jsx (hunk 2 above)
Comment (martindev727) [reply src/components/learn-more/learn-more.component.jsx]: well.
The font-size on mobile is 14px now

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx (hunk 5 above)
Comment (martindev727) [reply src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: fixed

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx (hunk 4 above)
Comment (martindev727) [reply src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: fixed

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx [hunk 8]
```diff
@@ -1,7 +1,148 @@
...
+        </div>
+      </div>
+      <div className="primary-container">
+        <div className="app-screening">
+          <div className="app-screeing-comment">
+            <div className="tertiary-header">
+              Two narcolepsy screening tools are now available in 1 app
+            </div>
+            <SiteLink className="link-text" to="Screener App">
+              Learn more about the Narcolepsy Screener App&nbsp;&nbsp;<FontAwesomeIcon className="arrow-right" icon="angle-right" />
```
Comment (rphovley) [inline src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: Less space between arrow and text based on designs
screen shot 2018-08-22 at 8 42 55 am

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx [hunk 9]
```diff
@@ -1,7 +1,148 @@
...
+      content:
+        'American Academy of Sleep Medicine. Central disorders of hypersomnolence. In: ',
+      source: 'The International Classification of Sleep Disorders – Third Edition (ICSD-3). ',
+      info: 'Darien, IL: American Academy of Sleep Medicine; 2014.'
+    }
+  ]
+  return (
+    <div className="swiss-narcolepsy-scale">
+      <div className="primary-container">
+        <TopLinkComponent topLinkContent={topLinkContent} />
```
Comment (rphovley) [inline src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: Top link should be flush with the right side of the page at mobile sizes
screen shot 2018-08-22 at 8 38 05 am

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx (hunk 8 above)
Comment (martindev727) [reply src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: Fixed

File: src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx (hunk 9 above)
Comment (rphovley) [reply src/components/swiss-narcolepsy-scale/swiss-narcolepsy-scale.component.jsx]: Looks like I forgot to be specific that it should be flush with the right side on desktop

**Pull request merged**